MetaMask/metamask-extension
# Fix #19647 - Allow importing multiple tokens

## Explanation

* Fixes #19647

This is a long-standing bug where trying to import multiple tokens at once didn't work, only the last would be added.  This PR fixes that issue.

## Screenshots/Screencaps

https://github.com/MetaMask/metamask-extension/assets/46655/5e1b6108-dd61-4a82-a4a2-94ec90cbac06

## Manual Testing Steps

1.  From the home screen, click "Import Tokens"
2. Search for and add multiple tokens
3. See all tokens added to the home screen

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
